### ISSUE TITLE
Unable to compile

### ISSUE THREAD

**Issue by oti-dev** [zcoinofficial/zcoin#196]
Okay, it's been like two weeks that the code from repo is not compilable. What happened, what's been messed up with tor integration?

1. https://github.com/zcoinofficial/zcoin/issues/193
2. https://github.com/zcoinofficial/zcoin/issues/188
3. https://github.com/zcoinofficial/zcoin/issues/191 (closed with no explanation - unsolved to me)

Is there going to be any comment on this? How to approach the issue? Are there new dependencies required now, which are not mentioned in the build instructions?

**Comment by oti-dev** [comment]
Missed it. Added now, thanks :)

**Comment by notatestuser** [comment]
yeah thanks for making this. we should not have to use precompiled binaries.

**Comment by devwarrior777** [comment]
It should build now with 0.13.5.8 as the latest. May we close this out please . . .
